src/pages/settings/Security/SecuritySettingsPage.js

```diff
@@ -47,11 +48,14 @@ function SecuritySettingsPage(props) {
     ];
 
     return (
-        <ScreenWrapper>
-            <HeaderWithBackButton
-                title={props.translate('initialSettingsPage.security')}
-                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
-            />
+        <IllustratedHeaderPageLayout
+            title={props.translate('initialSettingsPage.security')}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
+            shouldShowBackButton
+            shouldShowCloseButton
```

Comment:
I see this is in the design file but the Close buttons are deprecated with the new navigation framework so I am going to remove this

Reply:
https://github.com/Expensify/App/pull/28620